reana-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reana-ui",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
Should this instead be 0.9.2.1?

Reply:
We have been numbering components independently in the past, and some bring new features, as it were, so I think we can go with independent versioning of components.